modules/opm-cli-ref-serve.adoc

```diff
@@ -23,13 +23,28 @@ $ opm serve <source_path> [<flags>]
 |===
 |Flag |Description
 
+|`--cache-dir` (string)
+|If this flag is set, it syncs and persists the server cache directory.
+
+|`--cache-enforce-integrity`
+|Exits with an error, if the cache is not present or is invalidated. The default value is `true` when the `--cache-dir` flag is set and the `--cache-only` flag is `false`. Otherwise, the default is `false`.
```

Comment:
```suggestion
|Exits with an error if the cache is not present or is invalidated. The default value is `true` when the `--cache-dir` flag is set and the `--cache-only` flag is `false`. Otherwise, the default is `false`.
```